src/routes/pat/pat.ts

```diff
@@ -0,0 +1,60 @@
+import { sendError } from '@/errors';
+import { getUser, gqlSdk } from '@/utils';
+import { RequestHandler } from 'express';
+import Joi from 'joi';
+import { v4 as uuidv4 } from 'uuid';
+
+export const createPATSchema = Joi.object({
+  expiresAt: Joi.date().required(),
+  metadata: Joi.object(),
+}).meta({ className: 'CreatePATSchema' });
+
+export const createPATHandler: RequestHandler<
+  {},
+  {},
+  { refreshToken: string; metadata: object; expiresAt: Date }
```

Comment:
shouldn't `refreshToken` be an `accessToken`? In which case you'd just validate the signature and create a PAT for the user specified in the JWT